src/Uno.UI/UI/Xaml/Controls/Control/Control.cs

```diff
@@ -990,152 +1138,6 @@ protected virtual void OnBorderBrushChanged(Brush oldValue, Brush newValue)
-			}
-
-			if (GetIsEventOverrideImplemented(type, nameof(OnPointerCaptureLost), _pointerArgsType))
-			{
-				result |= RoutedEventFlag.PointerCaptureLost;
-			}
-
-			if (GetIsEventOverrideImplemented(type, nameof(OnPointerWheelChanged), _pointerArgsType))
-			{
-				result |= RoutedEventFlag.PointerWheelChanged;
-			}
-
-			if (GetIsEventOverrideImplemented(type, nameof(OnManipulationStarting), _manipStartingArgsType))
-			{
-				result |= RoutedEventFlag.ManipulationStarting;
-			}
-
-			if (GetIsEventOverrideImplemented(type, nameof(OnManipulationStarted), _manipStartedArgsType))
-			{
-				result |= RoutedEventFlag.ManipulationStarted;
-			}
-
-			if (GetIsEventOverrideImplemented(type, nameof(OnManipulationDelta), _manipDeltaArgsType))
-			{
-				result |= RoutedEventFlag.ManipulationDelta;
-			}
-
-			if (GetIsEventOverrideImplemented(type, nameof(OnManipulationInertiaStarting), _manipInertiaArgsType))
-			{
-				result |= RoutedEventFlag.ManipulationInertiaStarting;
-			}
-
-			if (GetIsEventOverrideImplemented(type, nameof(OnManipulationCompleted), _manipCompletedArgsType))
-			{
-				result |= RoutedEventFlag.ManipulationCompleted;
-			}
-
-			if (GetIsEventOverrideImplemented(type, nameof(OnTapped), _tappedArgsType))
-			{
-				result |= RoutedEventFlag.Tapped;
-			}
-
-			if (GetIsEventOverrideImplemented(type, nameof(OnDoubleTapped), _doubleTappedArgsType))
-			{
-				result |= RoutedEventFlag.DoubleTapped;
-			}
-
-			if (GetIsEventOverrideImplemented(type, nameof(OnRightTapped), _rightTappedArgsType))
-			{
-				result |= RoutedEventFlag.RightTapped;
-			}
-
-			if (GetIsEventOverrideImplemented(type, nameof(OnHolding), _holdingArgsType))
-			{
-				result |= RoutedEventFlag.Holding;
-			}
-
-			if (GetIsEventOverrideImplemented(type, nameof(OnDragEnter), _dragArgsType))
-			{
-				result |= RoutedEventFlag.DragEnter;
-			}
-
-			if (GetIsEventOverrideImplemented(type, nameof(OnDragOver), _dragArgsType))
-			{
-				result |= RoutedEventFlag.DragOver;
-			}
-
-			if (GetIsEventOverrideImplemented(type, nameof(OnDragLeave), _dragArgsType))
-			{
-				result |= RoutedEventFlag.DragLeave;
-			}
-
-			if (GetIsEventOverrideImplemented(type, nameof(OnDrop), _dragArgsType))
-			{
-				result |= RoutedEventFlag.Drop;
-			}
-
-			if (GetIsEventOverrideImplemented(type, nameof(OnKeyDown), _keyArgsType))
-			{
-				result |= RoutedEventFlag.KeyDown;
-			}
-
-			if (GetIsEventOverrideImplemented(type, nameof(OnKeyUp), _keyArgsType))
-			{
-				result |= RoutedEventFlag.KeyUp;
-			}
-
-			if (GetIsEventOverrideImplemented(type, nameof(OnLostFocus), _routedArgsType))
-			{
-				result |= RoutedEventFlag.LostFocus;
-			}
-
-			if (GetIsEventOverrideImplemented(type, nameof(OnGotFocus), _routedArgsType))
-			{
-				result |= RoutedEventFlag.GotFocus;
-			}
-
-			return ImplementedRoutedEvents[type] = result;
-		}
-
 		private static bool GetIsEventOverrideImplemented(Type type, string name, Type[] args)
```

Comment:
The existing reflection-based solution will likely return false for:

https://github.com/unoplatform/uno/blob/cd183d610dd800ee5a48af8008d72d8f8a16c40f/src/Uno.UI/UI/Xaml/Controls/ToggleSwitch.cs#L81

@carldebilly Can you confirm if this is the intended behavior?